src/styles/components/footer.less

```diff
@@ -7,6 +7,7 @@
 
   p {
     margin: 0 0 0.5em;
+    font-size: 12px;
```

Comment:
Suggest:
font-size: .875rem;
This is the same as the font down-sizing we use in the left-hand nav, and keeps proportional down-sizing should user scale fonts globally.

Reply:
Fixed with 5dbf8c7